gst/nnstreamer/include/nnstreamer_plugin_api_filter.h

```diff
@@ -183,14 +184,15 @@ typedef struct _GstTensorFilterFrameworkInfo
 {
   const char *name; /**< Name of the neural network framework, searchable by FRAMEWORK property. Subplugin is supposed to allocate/deallocate. */
   int allow_in_place; /**< TRUE(nonzero) if InPlace transfer of input-to-output is allowed. Not supported in main, yet */
-  int allocate_in_invoke; /**< TRUE(nonzero) if invoke_NN is going to allocate outputptr by itself and return the address via outputptr. Do not change this value after cap negotiation is complete (or the stream has been started). */
+  int allocate_in_invoke; /**< TRUE(nonzero) if invoke_NN is going to allocate outputptr by itself and return the address via outputptr. Do not change this value after cap negotiation is complete (or the stream has been started). In the case of dynamic invoke, allocate_in_invoke should be TRUE.*/
```

Comment:
Add a condition-check somewhere to enforce the rule, ```In the case of dynamic invoke, allocate_in_invoke should be TRUE.```.

LGTM other than this.
@jaeyun-jung or @again4you : please take a look and proceed.